src/components/DataExplorerApp/VizSettingsForm/index.tsx

```diff
@@ -39,6 +33,21 @@ export function VizSettingsForm({
     labelFn: getProp("displayName"),
   });
 
+  // helper to seed next config with cached XY
+  function seedXY<T extends VizConfig>(next: T): T {
```

Comment:
Don't use argument names like `next` it's always better to be more specific. Such as `newVizConfig`